packages/js/src/ui/components/PopoverTarget.tsx

```diff
@@ -0,0 +1,24 @@
+import { ParentComponent } from 'solid-js';
+import { useStyle } from '../helpers';
+import { usePopover } from './Popover';
+
+export const PopoverTarget: ParentComponent = (props) => {
+  const { setTargetRef, onToggle } = usePopover();
+  const style = useStyle();
+
+  return (
+    <button
+      ref={setTargetRef}
+      onClick={onToggle}
+      class={style(
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** Decided to have the PopoverTarget to have the bellContainer styles.‏

Reply:
⚠️ We shouldn't apply the styles here. Especially, the `bellContainer` class. Ideally, this should receive an `asChild` prop.